<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tituloCadastro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Geometr212 BkCn BT</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Geometr212 BkCn BT&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>CADASTRO ALUNOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tituloIdade">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>Idade:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tituloNome">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Geometr212 BkCn BT&quot;;
</string>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idade">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tituloEmail">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tituloDados">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Geometr212 BkCn BT</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Geometr212 BkCn BT&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 126, 184);</string>
    </property>
    <property name="text">
     <string>Dados Pessoais:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tituloCurso">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Geometr212 BkCn BT</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Geometr212 BkCn BT&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 126, 184);</string>
    </property>
    <property name="text">
     <string>Escolha o curso:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="javascript">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>Javascript</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="python">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>Python</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="php">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>PHP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoCadastroFinal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
font: 13pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
